README.md

```diff
@@ -1,50 +1,94 @@
 [![Build Status](https://travis-ci.org/kubernetes-csi/external-provisioner.svg?branch=master)](https://travis-ci.org/kubernetes-csi/external-provisioner)
 
-# Kubernetes external provisioner that works with CSI volumes
+# CSI provisioner
 
-This is an example external provisioner for Kubernetes which provisions using CSI Volume drivers..  It's under heavy development, so at this time README.md is notes for the developers coding.  Once complete this will change to something user friendly.
+The external-provisioner is a sidecar container that dynamically provisions volumes by calling `ControllerCreateVolume` and `ControlerDeleteVolume` functions of CSI drivers. It is necessary because internal persistent volume controller running in Kubernetes controller-manager does not have any direct interfaces to CSI drivers.
 
-# Build
+## Overview
+The external-provisioner is an external controller that monitors `PersistentVolumeClaim` objects created by user and creates/deletes volumes for them. Full design can be found at Kubernetes proposal at [container-storage-interface.md](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/storage/container-storage-interface.md)
 
-```bash
-make csi-provisioner
-```
+## Compatibility
 
-# Test
+This information reflects the head of this branch.
 
-## Start Kubernetes
+| Compatible with CSI Version                                                                | Container Image                        | Min K8s Version |
+| ------------------------------------------------------------------------------------------ | ---------------------------------------| --------------- |
+| [CSI Spec v1.0.0](https://github.com/container-storage-interface/spec/releases/tag/v1.0.0) | quay.io/k8scsi/csi-provisioner:v1.0.1  | 1.13            |
 
-Run a local kubernetes cluster built from latest master code
+## Feature status
 
-## Run Storage Provider
+Various external-provisioner releases come with different alpha / beta features. Alpha features are disabled by default and must be explicitly enabled on command line. Beta features are enabled by default and can be disabled on command line. Check `--help` output for alpha/beta features in each release.
 
-### Use HostPath drivers
+Following table reflects the head of this branch.
 
-Go to drivers and run:
+| Feature  | Status | Description                                                                                |
```

Comment:
Do we need a min K8s column?

Reply:
And a default column

Reply:
Added both, but it's tricky. We introduced CSINodeInfo in 1.13, but 1.13 is compatible only with sidecar 1.0.x, not with current head. For topology as it is implemented in 1.14, 1.14 is the minimal version.